tools/c7n_gcp/c7n_gcp/resources/loadbalancing.py

```diff
@@ -214,3 +214,132 @@ def get(client, resource_info):
             return client.execute_command('get', {
                 'project': resource_info['project_id'],
                 'healthCheck': resource_info['name']})
+
+
+@resources.register('loadbalancing-target-http-proxy')
```

Comment:
there's a consistent issue on the naming for these resources, ie. load balancing is the act / verb. load-balancer is the noun. load-balancer sounds like its a possessive of an instance which it's not. `loadbalance-` would be the prefix to indicate the intent which is service namespace.